core/scripts/cli/cli.js

```diff
@@ -1,11 +1,46 @@
-const cli = async function(callback) {
-  try {
-    console.log("You have started the UMA CLI!");
-  } catch (e) {
-    callback(e);
-  }
+const inquirer = require("inquirer");
+const vote = require("./vote");
+const wallet = require("./wallet");
 
-  callback();
+const collectInputs = async () => {
+  const prompts = [
+    {
+      type: "list",
+      name: "topMenu",
+      message: "Top level menu. What do you want to do?",
+      choices: ["wallet", "vote", "claim rewards", "DVM system status", "help", "exit"]
+    }
+  ];
+
+  return await inquirer.prompt(prompts);
 };
 
-module.exports = cli;
+async function run() {
+  let run = true;
+  while (run) {
+    const inputs = await collectInputs();
+    switch (inputs["topMenu"]) {
```

Comment:
nit: why not just return the "topMenu" field from the `collectInputs` function, so you don't have to touch the inquirer data structure outside of that function?